packages/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Link } from 'gatsby'
+import * as css from './Tabs.module.scss'
+import classnames from 'classnames'
+import { TextCaps } from '@elvia/ui'
+import {
+  HddArrowRight,
+  HddArrowLeft,
+} from '@hafslundnett/hdd-icons/hdd-icons-react'
+
+interface LinkItem {
+  name: string
+  url: string
+}
+
+export interface TabsProps {
+  className?: string
+  tabs?: LinkItem[]
+  uri?: string
+}
+
+const useWindowDimensions = () => {
+  const [width, setWidth] = React.useState(window.innerWidth)
+
+  const updateWidth = () => {
+    setWidth(window.innerWidth)
+  }
+
+  useEffect(() => {
+    window.addEventListener('resize', updateWidth)
+    return () => window.removeEventListener('resize', updateWidth)
+  })
+
+  return {
+    width,
+  }
+}
+
+export const Tabs: React.FC<TabsProps> = ({ tabs, uri }) => {
+  const [arrow, setHideArrows] = useState<string>()
```

Comment:
Her sier du at arrow alltid vil være av typen string, men du setter ingen default string verdi så den vil være null eller undefined ut av boksen.

```suggestion
  const [arrow, setHideArrows] = useState<string>('')
```

Reply:
Dette burde vært catchet av es-lint så hvis du ikke får feil i vscode eller under commit så er det noe som ikke stemmer helt.

Reply:
Har ikke fått noen feil nei.